internal/distro/rhel8/distro.go

```diff
@@ -1780,6 +1780,14 @@ func newDistro(distroName string) distro.Distro {
 				},
 			},
 		},
+		GCPGuestAgentConfig: &osbuild.GcpGuestAgentConfigOptions{
+			GcpGuestAgentConfigScope: "distro",
```

Comment:
You should use `GcpGuestAgentConfigScopeDistro` instead of the string literal.

Reply:
Nicely seen, thanks!